src/renderer/containers/Header/UserMenu.tsx

```diff
@@ -28,8 +25,9 @@ export const UserMenu: React.FC<{
   const refreshPlayKey = useAccount((store) => store.refreshPlayKey);
   const loading = useAccount((store) => store.loading);
   const [openLogoutPrompt, setOpenLogoutPrompt] = React.useState(false);
-  const theme = useTheme();
-  const fullScreen = useMediaQuery(theme.breakpoints.down("xs"));
+  const [name, setName] = React.useState(user.displayName);
```

Comment:
This shouldn't be a local React state. Since it means that other components that would want to render a user profile name would be out of date. `setDisplayName` should be a new method in the `useAccount` hook so that when updated all instances of the name get updated.